qiskit/assembler/assemble_schedules.py

```diff
@@ -17,25 +17,23 @@
 from typing import Any, Dict, List, Tuple
 import hashlib
 
+from qiskit import qobj
+from qiskit import pulse
+from qiskit.assembler.run_config import RunConfig
 from qiskit.exceptions import QiskitError
-from qiskit.pulse import Schedule, Acquire, Delay, Play, reschedule
-from qiskit.pulse.pulse_lib import ParametricPulse, SamplePulse
-from qiskit.pulse.commands import (Command, PulseInstruction, AcquireInstruction,
-                                   DelayInstruction, ParametricInstruction)
-from qiskit.qobj import (PulseQobj, QobjHeader, QobjExperimentHeader,
-                         PulseQobjInstruction, PulseQobjExperimentConfig,
-                         PulseQobjExperiment, PulseQobjConfig, PulseLibraryItem)
-from qiskit.qobj.converters import InstructionToQobjConverter, LoConfigConverter
+from qiskit.pulse import instructions
+from qiskit.pulse import transforms
+from qiskit.pulse import pulse_lib
+from qiskit.pulse import commands
+from qiskit.qobj import converters
+from qiskit.qobj import utils as qobj_utils
```

Comment:
@ajavadia Is this the new style you want to follow throughout Qiskit?

Reply:
I'm not suggesting this go throughout Qiskit (although in this sort of situation I think it does make it clearer). This was an attempt to mitigate circular dependencies that were causing Python to error out as at this state only importing the module does not put any demands on having loaded specific objects within the module.

Reply:
Yeah this is fine.

Reply:
Fwiw, I proposed doing this a long time ago for the same reasons: https://github.com/Qiskit/qiskit-terra/issues/1170 and it was decided not to refactor everything.

Reply:
I'm fine whatever you guys decide is best.

Reply:
Ok, I'm going to keep this at least for the work here. I will likely also update all pulse related modules in the future. For the most part we have chosen names that work with their namespace, eg., `pulse.Schedule`.